cns/service/main.go

```diff
@@ -795,6 +801,35 @@ func main() {
 		}
 	}
 
+	type CNSService struct {
+		pb.UnimplementedCNSServiceServer
+		Logger *zap.Logger
+	}
+
+	// Define gRPC server settings
+	settings := grpc.GrpcServerSettings{
+		IPAddress: "::",
+		Port:      8080,
+	}
+
+	// Initialize CNS service
+	cnsService := &CNSService{Logger: z}
+
+	// Create a new gRPC server
+	server, err := grpc.NewServer(settings, cnsService, z)
+	if err != nil {
+		logger.Errorf("Could not initialize gRPC server: %v", err)
+		os.Exit(1)
+	}
+
+	// Start the gRPC server
+	go func() {
```

Comment:
nitpick: lets move all of the gRPC server initialization code to right above line 907, which is where the HTTP server is started up

Reply:
I have moved all the gRPC code above the HTTP server start up above line 907